src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -46,17 +50,25 @@ class DBSCAN {
      */
     vector<unsigned int>* _neighbors;
 
-    int _min_neighbors = 5;
-    float _radius      = 5;
-
+    int _min_neighbors               = 5;
+    float _radius                    = 5;
     unsigned int _sequential_cut_off = 1000;
+    Plane _plane;
 
-  public:
     /*
      * Constructor:
      * Takes in minimum number of neighbours and radius as parameters
+     * as well as plane (XY or YZ)
      */
-    DBSCAN(int min_neighbours = 5, float radius = 5);
+    DBSCAN(int min_neighbours = 5, float radius = 5, Plane plane = XY);
+    ~DBSCAN() { delete[] this->_neighbors; }
+
+  public:
+    static vector<pcl::PointCloud<pcl::PointXYZ>>
+    getClusters(pcl::PointCloud<pcl::PointXYZ>::Ptr pcl_ptr,
+                int min_neighbours = 5,
+                float radius       = 5,
+                Plane plane        = XY);
```

Comment:
I'm ok with setting default values in the constructor, but I'd rather we explicitly pass them instead of also having default values for method arguments. This helps prevent errors like "that's weird, I changed `min_neighbours` in my call to the constructor, but it's not doing anything....".